packages/common/src/resolve.js

```diff
@@ -0,0 +1,12 @@
+function resolve(obj, path) {
+  try {
+    return path
+      .split('.')
+      .reduce((prev, curr) => prev && prev[curr], obj)
+      .toString();
```

Comment:
Why do we need to convert it to a string?

Reply:
We're enforcing `path` to be an object property, not an array position

Reply:
I've replaced the entire resolve for [`lodash.get`](https://lodash.com/docs/4.17.15#get)

But we could also use [`just-safe-get`](https://www.npmjs.com/package/just-safe-get), as suggested by @leonardokl 

What do you think @tcK1?

Reply:
I really liked this `just-safe-get` package! Maybe we could bench them and see if there's a difference (Since these functions will probably be called sometimes).

But all in all, both choices seems nice!